Mark9Mbugua/Politico-V
# #163933811 Write tests that fail

#### What does this PR do?
It implements TDD by writing tests that fail at first
#### Description of Task to be completed?
- Write tests on separate tests files that initially fail
- Configure the test database properly. The tests should simply fail due to the lack of implementation of 
 the endpoints' code but should not encounter any errors

#### How should this be manually tested?
- Clone this repo in a local repo
- Navigate to the root directory
- Install pytest
- On the terminal run pytest
- Observe if the tests fail

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2252715
#163933811
